workshop/nano1-vuex.md

```diff
@@ -0,0 +1,281 @@
+# 👩‍🎓 Nano Activity (number): Title of Activity
+
+| **Project Goal**            | What we're going to build                                                                                                                                   |
+| --------------------------- | ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------ |
+| **What you’ll learn**       | *Vuex* is a state management library made for Vue, in this nano you will learn the basic concepts, and how to use it on a Vue application.
+| **Tools you’ll need**       | A modern browser like Chrome. Access to [CodeSandbox](https://codesandbox.io). [Vue Developer tools](https://chrome.google.com/webstore/detail/vuejs-devtools/nhdogjmejiglipccpnnnanhbledajbpd?hl=en) extension for Chrome.
+| **Time needed to complete** | 10-30 minutes
+
+
+# We're going to create a Videogame Card Collection using Vuex
+
+In this activity we will create a simple collection of card of some classic video game titles that will be powered by a Vuex state.
+## 1. Get Started
+
+Head over to our CodeSandbox at [![Edit Vuex 101 Nano - Start](https://codesandbox.io/static/img/play-codesandbox.svg)](https://codesandbox.io/s/ry03r15o9o)
+
+There are three important files at play:
+
+1. `App.vue` Our main file with some boilerplate code for looping through an array of games:
+```html
+<div class="col-12 col-md-6 col-lg-4" v-for="game in games">
+  <card
+    @click="removeGame(game);"
+    :name="game.title"
+    :image="game.img"
+  ></card>
+</div>
+
+<ul class="list-group">
+  <li class="list-group-item" v-for="title in gameTitles" :key="title">
+    {{ title }}
+  </li>
+</ul>
+```
+
+2. `components/Card.vue` This will present our game information in a card. It will receive a `name` and `image` properties.
+
+3. `data/data.json` Our mock-up games database, JSON format.
+
+## 2. Step 1 - Installing Vuex
+
+Since we are using codepen, the only thing we need to install Vuex is to add it to our list of 
+dependencies, so go to the `Dependencies` button on the bottom left of your screen and add it to
+our project.
+
+::: tip 💡
+When working on a local project, you should first add the dependency with `npm install vuex` or
```

Comment:
Awesome point for people who want to work not only in sandboxes! Must-have for all `advanced` nanos :+1: